src/less/cta-button/cta-button.less

```diff
@@ -58,22 +58,32 @@ span.cta-btn__cell--fixed-height svg.icon {
     width: 1rem;
 }
 
+a.cta-btn--truncated {
+    .btn-truncate();
+}
+
 a.cta-btn--large {
-    border-radius: calc(@button-height-large / 2);
+    border-radius: @button-border-radius-large;
+    display: inline-flex;
     font-size: @font-size-16;
     min-height: @button-height-large;
     padding: @button-padding-vertical-large @button-padding-horizontal;
 }
 
+// DEPRECATED remove cta-btn--large-truncated in next major version
 a.cta-btn--large-truncated {
```

Comment:
@ArtBlue yes, I added them just to avoid breaking changes. Instead of padding, we are using ` display: inline-flex;` to achieve the alignment. Moving forward, the idea is to use `a.cta-btn--truncated` along with `a.cta-btn--large` to achieve truncation for large buttons

Reply:
This was the initial suggestion from the issue.